### PR TITLE
feat(theme): Add new blues and use internally, darken sk-black

### PR DESCRIPTION
**Pull request by markdalgleish**
This adds the following colours:
- `@sk-blue-darker`
- `@sk-blue-dark`
- `@sk-blue-light`
- `@sk-blue-lighter`

The new blues have been applied to links, buttons and form accents.

For accessibility reasons, `@sk-black` has also been darkened slightly.

While revisiting form elements, some visual styling has been tweaked:

- Button shadows are now inset, ensuring they match the button hue
- SlideToggle animations have been polished up a bit

### CODE REVIEW

File: react/Autosuggest/Autosuggest.less [hunk 1]
```diff
@@ -49,7 +49,7 @@
   line-height: @sectionTitleHeight;
   height: @sectionTitleHeight;
   margin: @row-height (@gutter-width * 0.75) 0;
-  color: darken(@sk-highlight, 10%);
+  color: @sk-blue-lighter;
```
Comment (michaeltaranto) [inline react/Autosuggest/Autosuggest.less]: Thoughts on making this `@sk-form-accent`?

Comment (markdalgleish) [reply react/Autosuggest/Autosuggest.less]: Yeah, that's a good idea.

Comment (markdalgleish) [reply react/Autosuggest/Autosuggest.less]: Done.

File: react/Autosuggest/Autosuggest.demo.js [hunk 2]
```diff
@@ -94,6 +94,22 @@ export default {
             className: classnames(className, styles.rootFocus)
           })
         },
+        {
+          label: 'Grouped',
+          transformProps: ({ autosuggestProps, ...props }) => ({
+            ...props,
+            autosuggestProps: {
+              ...autosuggestProps,
+              multiSection: true,
```
Comment (markdalgleish) [inline react/Autosuggest/Autosuggest.demo.js]: Realised that I had no way to preview the updated section heading colour, so took this as a good opportunity to add this to our demo page 😄🙌

File: docs/src/components/Home/Hero/ButtonsPreview/ButtonsPreview.js [hunk 3]
```diff
@@ -9,14 +9,14 @@ export default function ButtonsPreview() {
   return (
     <div className={styles.root}>
       <div className={styles.group}>
-        <Button color="pink" className={styles.button}>Normal</Button>
+        <Button color="pink" className={styles.button}>Button</Button>
         <Button color="pink" className={classnames(styles.button, buttonStyles.rootHover)}>Hover</Button>
-        <Button color="pink" className={classnames(styles.button, buttonStyles.rootActive)}>Pressed</Button>
+        <Button color="pink" className={classnames(styles.button, buttonStyles.rootActive)}>Active</Button>
```
Comment (markdalgleish) [inline docs/src/components/Home/Hero/ButtonsPreview/ButtonsPreview.js]: Changed this to `Active`, because `Pressed` actually rendered as `Press...`

**Pull request merged**